services/stream-functions/serverless.yml

```diff
@@ -31,49 +31,9 @@ custom:
       - val
       - prod
   tableStreamArn: ${cf:database-${self:custom.stage}.AmendmentsTableStreamArn}
-  elasticSearchDomainEndpoint: ${cf:elasticsearch-${self:custom.stage}.ElasticSearchDomainEndpoint}
-  elasticSearchDomainArn: ${cf:elasticsearch-${self:custom.stage}.ElasticSearchDomainArn}
   sesSourceEmailAddress: ${ssm:/configuration/${self:custom.stage}/sesSourceEmailAddress~true, ssm:/configuration/default/sesSourceEmailAddress~true, "admin@example.com"}
   reviewerEmailAddress: ${ssm:/configuration/${self:custom.stage}/reviewerEmailAddress~true, ssm:/configuration/default/reviewerEmailAddress~true, "reviewteam@example.com"}
 
-functions:
-  elasticsearchIndexer:
-    handler: handlers/elasticsearchIndexer.handler
-    events:
-      - stream:
-          arn: ${self:custom.tableStreamArn}
-          startingPosition: LATEST
-          maximumRetryAttempts: 2
-    role: LambdaElasticSearchIndexerRole
-    environment:
-      ES_ENDPOINT_URL: https://${self:custom.elasticSearchDomainEndpoint}
-      INDEX: amendments
-      TYPE: json
-    maximumRetryAttempts: 2
-  emailSubmitter:
```

Comment:
This function should not be removed.  It isn't related to elasticsearch.

Reply:
lines 40-52 is the only function that should be removed